lib/array_inject.rb

```diff
@@ -0,0 +1,10 @@
+class SubclassArray < Array
+
+	def injecting (array,sum)
```

Comment:
Use 2 (not 1) spaces for indentation.<br>Space missing after comma.<br>Do not put a space between a method name and the opening parenthesis.<br>Tab detected.